Services/Component/classes/class.ilPlugin.php

```diff
@@ -606,30 +606,21 @@ public function addBlockFile($a_tpl, $a_var, $a_block, $a_tplname)
 	 * @param $a_slot_id
 	 * @param $a_pname
 	 *
-	 * @description Create plugin record, if not existing
```

Comment:
Because i droped the check of existing. This function only get called after a previous check the plugin record does not exist.